docs/USAGE-GUIDE.md

```diff
@@ -535,6 +544,58 @@ underlying encrpyt/decrypt functions are not available in older versions.
+|   "20190214"   |      X       |         X       |      X        |          |
+|   "20170718"   |      X       |         X       |      X        |          |
+|   "20170405"   |      X       |         X       |      X        |          |
+|   "20170328"   |      X       |         X       |      X        |          |
+|   "20170210"   |      X       |         X       |      X        |          |
+|   "20160824"   |      X       |         X       |      X        |          |
+|   "20160804"   |      X       |         X       |      X        |          |
+|   "20160411"   |      X       |         X       |      X        |          |
+|   "20150306"   |      X       |         X       |      X        |          |
+|   "20150214"   |      X       |         X       |      X        |          |
+|   "20150202"   |      X       |         X       |      X        |          |
+|   "20141001"   |      X       |         X       |      X        |          |
+|   "20140601"   |      X       |         X       |      X        |          |
+|   "20190120"   |      X       |         X       |      X        |          |
+|   "20190121"   |      X       |         X       |      X        |          |
+|   "20190122"   |      X       |         X       |      X        |          |
+| "default_tls13"|      X       |         X       |      X        |    X     |
+|   "20190801"   |      X       |         X       |      X        |    X     |
+|   "20190802"   |      X       |         X       |      X        |    X     |
+|   "20200207"   |      X       |         X       |      X        |    X     |
+
+The following chart maps the security policy version to the supported curves/groups: 
+
+|    version     |   secp256r1  |  secp384r1 | x25519 | 
+|----------------|--------------|------------|--------|
+|   "default"    |      X       |      X     |        | 
+|   "20190214"   |      X       |      X     |        | 
+|   "20170718"   |      X       |      X     |        | 
+|   "20170405"   |      X       |      X     |        | 
+|   "20170328"   |      X       |      X     |        | 
+|   "20170210"   |      X       |      X     |        | 
+|   "20160824"   |      X       |      X     |        | 
+|   "20160804"   |      X       |      X     |        | 
+|   "20160411"   |      X       |      X     |        | 
+|   "20150306"   |      X       |      X     |        | 
+|   "20150214"   |      X       |      X     |        | 
+|   "20150202"   |      X       |      X     |        | 
+|   "20141001"   |      X       |      X     |        | 
+|   "20140601"   |      X       |      X     |        | 
+|   "20190120"   |      X       |      X     |        | 
+|   "20190121"   |      X       |      X     |        | 
+|   "20190122"   |      X       |      X     |        | 
+| "default_tls13"|      X       |      X     |   X    | 
+|   "20190801"   |      X       |      X     |   X    | 
+|   "20190802"   |      X       |      X     |   X    | 
```

Comment:
It looks like this uses s2n_ecc_preferences_20140601 which doens't have x25519 support

Reply:
Updating, good catch.